src/LiveComponent/doc/index.rst

```diff
@@ -1809,7 +1809,7 @@ Now, create a Twig component to render the form::
     use Symfony\UX\LiveComponent\DefaultActionTrait;
     use Symfony\UX\LiveComponent\LiveCollectionTrait;
 
-    #[AsLiveComponent]
+    #[AsLiveComponent('blog-post-collection-type')]
```

Comment:
You can revert this. By *not* passing a name (that's the recommendation), the name becomes the class name, so `BlogPostCollectionType`. 

But also, isn't `BlogPostCollectionType` kind of a bad name for this component? It is a component to render the blog post form - so perhaps we should rename the class to `BlogPostForm`?